lib/strava/errors/fault.rb

```diff
@@ -2,15 +2,17 @@ module Strava
   module Errors
     class Fault < ::Faraday::ClientError
       def message
-        response[:body]['message'] || super
+        response[:body]['message'] || response[:body]['status'] || super
       end
 
       def headers
         response[:headers]
       end
 
       def errors
-        response[:body]['errors']
+        return response[:body]['errors'] if response[:body]['errors']
+
+        [response[:body]['error']] if response[:body]['error']
```

Comment:
I wrap the error in an array so that it stays consistent with normal error format